roles/lib_openshift/src/doc/sdnvalidator

```diff
@@ -0,0 +1,27 @@
+# flake8: noqa
+# pylint: skip-file
```

Comment:
This is a Python file, shouldn't its name end with `.py`?
I see none of the other files in this directory have an extension, @kwoodson was there some idea behind that?

Sad to see that `lib_openshift` simply bypasses both flake8 and pylint...